cart/domain/cart/cartServicePorts.go

```diff
@@ -87,9 +87,13 @@ type (
 
 	// ItemUpdateCommand defines the update item command
 	ItemUpdateCommand struct {
-		// Source Id of where the items should be initial picked - This is set by the SourcingLogic
-		SourceID       *string
-		Qty            *int
+		// SourceID of where the items should be initial picked - This is set by the SourcingLogic
+		SourceID *string
+		// ExternalSourceID contains the picking location used by an external system
+		ExternalSourceID *string
```

Comment:
SourceId stays the unique key for a Source - right?
So it feels unlogical to add an additional property here since ExternalSourceId will always be determined by the SourceId.

Reply:
We even discussed to move sourcing code completely from cart package - it can be done by the CartAdapters
(PlaceOrderAdapter OMS is doing sourcing himself / Magento Adapter can do it also)

Reply:
> SourceId stays the unique key for a Source - right?
> So it feels unlogical to add an additional property here since ExternalSourceId will always be determined by the SourceId.

Good point, I guess we'll then use a field in the `AdditionalData` property.

Reply:
> We even discussed to move sourcing code completely from cart package - it can be done by the CartAdapters
> (PlaceOrderAdapter OMS is doing sourcing himself / Magento Adapter can do it also)

yeah, maybe something we can do for v4 of flamingo-commerce, i'll add an issue.